table-inheritance/README.md

```diff
@@ -0,0 +1,51 @@
+# Table Inheritance
+
+**Category:** Data Access Pattern, Structural Pattern
+
+**Tags:** Decoupling, Inheritance, Polymorphism, Object Mapping, Persistence, Data Transformation
```

Comment:
The format of README.md is very specific and should be followed precisely. See instructions here: https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute